webapp/.eslintrc.json

```
@@ -632,5 +632,47 @@
         "onlyEquality": false
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.tsx", "**/*.ts"],
+      "extends": [
```

Here we can replace it directly with the typescript plugin setting.
```suggestion
      "extends": "plugin:@typescript-eslint/recommended",
```

---

done